content/src/main/content/jcr_root/apps/acs-commons/cloudservices/pwa/clientlibs/clientlib-idb/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    jcr:title="ACS Commons IndexedDb"
+    allowProxy="{Boolean}true"
+    categories="[acs-commons.idb]"/>
```

Comment:
IIRC we mark vendor resources in vendor folders (skips code quality checks) and also add vendor to the category so its clear we (acs commons) arent providing this JS.